### PR TITLE
Update development.md

### PR DESCRIPTION
**Pull request by Harshdev098**
### Why:

Closes: 

### What's being changed (if available, include any code snippets, screenshots, or gifs): 
 the readme link is not working in development.md file

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: contributing/development.md
Comment (Justin09111974) [inline contributing/development.md:1]: branch

Comment (Justin09111974) [reply contributing/development.md:1]: branch

Comment (Harshdev098) [reply contributing/development.md:1]: Main

File: contributing/development.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ For more info about working with this site, check out these READMEs:
 - [data/README.md](../data/README.md)
 - [data/reusables/README.md](../data/reusables/README.md)
 - [data/variables/README.md](../data/variables/README.md)
-- [includes/liquid-tags/README.md](../includes/liquid-tags/README.md)
+- [tests/fixtures/README.md](tests/fixtures/README.md)
```
Comment (steveward) [inline contributing/development.md:72]:
```suggestion
- [src/content-render/README.md](src/content-render/README.md)
```

Comment (steveward) [reply contributing/development.md:72]: This should be the correct link to the `content-render` section that discusses liquid tags.

File: contributing/development.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Today it's a dynamic Node.js webserver powered by Express, using [middleware](..
 
 The tooling for this site has changed over the years, but many of the tried-and-true authoring conventions of the original Jekyll site have been preserved:
 
-- Content is written in Markdown files, which live in the `content` directory.
+- Content is written in Markdown files, which are live in the `content` directory.
```
Comment (steveward) [inline contributing/development.md:57]:
```suggestion
- Content is written in Markdown files, which live in the `content` directory.
```
The current wording here is correct, so I'm going to remove the "are" from this line.

**Review by steveward — CHANGES_REQUESTED** [review]
Thanks for finding this broken link @Harshdev098! I found the correct link to the README that describes liquid tags, so I've added a suggestion with the correct link.

**Review by steveward — APPROVED** [review]
Thanks!

**Pull request merged**